deploy/apparmor/virtlet

```diff
@@ -29,9 +33,12 @@ profile virtlet flags=(attach_disconnected) {
   /{usr/,}sbin/ebtables rix,
   /{usr/,}sbin/brctl rix,
   /opt/cni/bin/calico* rix,
+  /opt/cni/bin/genie rix,
```

Comment:
What about other plugins, like flannel?
Same with ipam plugins (host-local, dhcp).
Maybe it needs a note in readme how to enable/disable particular plugins?

Reply:
For flannel it works, can you please provide a list of binaries specific to ipam plugins?

Reply:
So `/opt/cni/bin/flannel` (and probably the same will be with `/opt/cni/bin/bridge`) does not require such entry with `rix` flag, right?
It will have read right to `/run/flannel/subnet.env`?

Mainly used ipam plugin will be probably `/opt/cni/bin/host-local`, which stores own data files in `/var/lib/cni/networks/$NETWORK_NAME` - with network name defined as in plugin config file.

Reply:
I think it's better to provide access to the main binaries anyway, if there's a possibility that they will be executed. Will update rules